packages/schemas/src/index.ts

```diff
@@ -47,6 +47,32 @@ export function getJsonSchemaByCoreSchemaType(coreSchemaType: CoreSchemaType) {
 	}
 }
 
+export function getDefaultValueByCoreSchemaType(coreSchemaType: CoreSchemaType) {
+	switch (coreSchemaType) {
+		case CoreSchemaType.String:
+			return "";
+		case CoreSchemaType.Boolean:
+			return undefined;
+		case CoreSchemaType.Vector3:
+			return [0, 0, 0];
+		case CoreSchemaType.DateTime:
+			return undefined;
+		case CoreSchemaType.Email:
+			return "";
+		case CoreSchemaType.URL:
+			return "";
+		case CoreSchemaType.MemberId:
+			return "";
+		case CoreSchemaType.FileUpload:
+			return undefined;
+		case CoreSchemaType.Null:
+			return undefined;
```

Comment:
Looks like this needs cases for the new elements that I merged in yesterday! I would have added them but I wasn't totally sure what was best for number!

Reply:
ah yeah good question... I experimented a bit and it seems like `undefined` actually works okay here! also I got things mixed up in my [earlier comment](https://github.com/pubpub/platform/pull/687#discussion_r1790483864)—we're actually okay on controlled vs uncontrolled state here, but this helps such that when we validate, we don't get a whole sea of red because our string fields have undefined values

(if we don't have a default value for String here, then "title" and "content" immediately get marked as invalid which is alarming, as shown in this vid)


https://github.com/user-attachments/assets/8d7f4537-09aa-451d-af67-dba5c811e8e0

Reply:
ah right, makes sense! i've seen that on some of our other forms!